src/modules/General/components/fileUploaderMultiple/useFileUploaderMultiple.tsx

```diff
@@ -0,0 +1,123 @@
+import { useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+import { useTranslation } from 'react-i18next';
+import jpg from 'src/assets/icons/file-jpg.svg';
+import pdf from 'src/assets/icons/file-pdf.svg';
+import png from 'src/assets/icons/file-png.svg';
+import { AdaptorRes, UploadMediaRes, uploadMediaWithProgressAdaptor } from 'src/core/adaptors';
+
+export const useFileUploader = (
+  fileTypes: string[],
+  maxFileNumbers: number,
+  maxSize: number,
+  uploaded: UploadMediaRes[],
+  setUploaded: (newVal: UploadMediaRes[]) => void,
+  setShowFiles?: (files: File[]) => void,
+  showFiles?: File[],
+) => {
+  const { t: translate } = useTranslation();
+  const [error, setError] = useState('');
+  const [uploading, setUploading] = useState(false);
+  const [progress, setProgress] = useState(0);
+  const [files, setFiles] = useState<File[]>(showFiles || []);
+
+  const getAcceptedFileTypes = () => {
+    const types = [
+      { key: 'doc', doc: 'application/msword', extention: ['.doc'] },
+      {
+        key: 'docx',
+        doc: 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+        extention: ['.docx'],
+      },
+      { key: 'pdf', doc: 'application/pdf', extention: ['.pdf'] },
+      { key: 'svg', doc: 'image/svg+xml', extention: ['.svg'] },
+      { key: 'png', doc: 'image/png', extention: ['.png'] },
+      { key: 'jpg', doc: 'image/jpeg', extention: ['.jpg'] },
+      { key: 'gif', doc: 'image/gif', extention: ['.gif'] },
+    ];
+    const lowercaseFileTypes = fileTypes.map(t => {
+      return t.toLowerCase();
+    });
+
+    const acceptedTypes = {};
+    types.forEach(item => {
+      if (lowercaseFileTypes.includes(item.key)) acceptedTypes[item.doc] = item.extention;
+    });
+    return acceptedTypes;
+  };
+
+  const getSubtitle = () => {
+    let text = fileTypes.slice(0, fileTypes.length - 1).join();
+    text = `${text} or ${fileTypes[fileTypes.length - 1]} (max. ${maxSize}mb)`;
+    return text;
+  };
+
+  const readableFileSize = (fileSize: number) => {
+    const sizeInKb = fileSize / 1024;
```

Comment:
can you create a constant for `1024` to change it later easily and use it all through this file?